addons/website_event_track_quiz/static/src/xml/quiz_templates.xml

```diff
@@ -27,39 +27,63 @@
                                 <span t-esc="answer.text_value"/>
                             </a>
                         </t>
-                        <div class="o_quiz_quiz_answer_info list-group-item list-group-item-info d-none">
-                            <i class="fa fa-info-circle"/>
-                            <span class="o_quiz_quiz_answer_comment"/>
-                        </div>
                     </div>
                 </div>
-                <div t-if="!widget.track.completed" class="o_quiz_js_quiz_validation border-top pt-3"/>
-                <div t-else="" class="row">
-                    <div class="o_quiz_js_quiz_validation col py-2 bg-100 mb-2 border-bottom"/>
-                </div>
             </div>
         </div>
     </t>
 
+    <t t-name="quiz.badge">
+        <span class="badge badge-warning ml-2">
+            <t t-if="answer.awarded_points > 1">+ <t t-esc="answer.awarded_points"/> Points</t>
```

Comment:
Can we do:
```
<span>+ <t t-esc="answer.awarded_points"/></span>
<span t-if="answer.awarded_points == 1">Point</span>
<span t-else="">Points</span>
```

I'm a little worried for translations otherwise.